qiskit_experiments/library/tomography/qpt_experiment.py

```diff
@@ -52,50 +53,69 @@ class ProcessTomography(TomographyExperiment):
     def __init__(
         self,
         circuit: Union[QuantumCircuit, Instruction, BaseOperator],
+        backend: Optional[Backend] = None,
+        physical_qubits: Optional[Sequence[int]] = None,
         measurement_basis: basis.MeasurementBasis = basis.PauliMeasurementBasis(),
+        measurement_indices: Optional[Sequence[int]] = None,
         measurement_qubits: Optional[Sequence[int]] = None,
         preparation_basis: basis.PreparationBasis = basis.PauliPreparationBasis(),
+        preparation_indices: Optional[Sequence[int]] = None,
         preparation_qubits: Optional[Sequence[int]] = None,
         basis_indices: Optional[Iterable[Tuple[List[int], List[int]]]] = None,
         qubits: Optional[Sequence[int]] = None,
+        analysis: Optional[Union[BaseAnalysis, None]] = "default",
     ):
         """Initialize a quantum process tomography experiment.
 
         Args:
             circuit: the quantum process circuit. If not a quantum circuit
                 it must be a class that can be appended to a quantum circuit.
+            backend: The backend to run the experiment on.
+            physical_qubits: Optional, the physical qubits for the initial state circuit.
+                If None this will be qubits [0, N) for an N-qubit circuit.
             measurement_basis: Tomography basis for measurements. If not specified the
                 default basis is the :class:`~basis.PauliMeasurementBasis`.
-            measurement_qubits: Optional, the qubits to be measured. These should refer
-                to the logical qubits in the state circuit. If None all qubits
-                in the state circuit will be measured.
+            measurement_indices: Optional, the physical_qubit indices to be measured.
```

Comment:
physical_qubits with an s. (also consider double ticks here and elsewhere)

This comment applies to all three files.